utils/ANLEra_extra_ai_units.cfg

```diff
@@ -1,5 +1,72 @@
 #textdomain wesnoth-ANLEra
 
+#define ANL_AI_FACTION SIDE FACTION THEN
+	[switch]
+		variable={SIDE}.controller
+
+		[case]
+			value=ai
+			[switch]
+				variable={SIDE}.faction
+				[case]
+					value={FACTION}
+					{THEN}
+				[/case]
+			[/switch]
+		[/case]
+	[/switch]
+#enddef
```

Comment:
two switch vases with each only having one case?

Reply:
In case it becomes relevant to you, I want to mention that 1.13 added an [elseif] statement:
https://wiki.wesnoth.org/ConditionalActionsWML#.5Bif.5D

While one may wonder that the [elseif] statement accepts only [then] as subtag, one can write multiple [elseif] tags or an [else] tag to have the same behaviour.

Reply:
That’s the way it was done before I tried to simplify/clean it.
I kept it as is because I was unsure about how conditionals worked in Wesnoth (and lost a lot of time on other, unexpected issues before I forgot about it).

But yeah, it should probably be fixed. My bad. :/